ISISComputingGroup/ibex_gui
# Update OPI Based on IOC Refactor

### Description of work

Updated the OPI field values to reflect changes made in [Db refactor](https://github.com/ISISComputingGroup/EPICS_PearlPressure/pull/3) in EPICS_PearlPressure repository so that the OPI displays PV values

### Acceptance criteria

- [ ] PEARLPC IOC OPI displays correct values in appropriate fields for changes made in [Db refactor](https://github.com/ISISComputingGroup/EPICS_PearlPressure/pull/3) as were previously displayed before changes were made to IOC.

### Documentation

* [WIKI](https://github.com/ISISComputingGroup/IBEX/wiki/PEARL-Instrument-Details#pearl-notes)
* IOC PR that changes made in this PR are for: [Db_refactor](https://github.com/ISISComputingGroup/EPICS_PearlPressure/pull/3)
* Testing instructions:
    * Checkout [main](https://github.com/ISISComputingGroup/EPICS_PearlPressure) in `C:\Instrument\Apps\EPICS\support\PearlPressureController\master` and run `make` to ensure they are up to date.
    * Run the tests with `- a` flag to run IOC in simulation mode: `system_tests\run_tests.bat -a`
    * Checkout this PR's branch `opi_refactor` in `C:\Instrument\Dev\ibex_gui`
    * Open `ibex.product` in Eclipse IDE and launch IBEX client after first cleaning project and potentially reloading `targetplatform.target` if needed.
    * From launched IBEX client, add a new device screen for PEARLPC OPI and check OPI behaves as desired by playing around with the controls and current settings, checking that values are populated when set. 
---

#### Code Review
